assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+const API_KEY = "b248c5c1ee789732d38aa9fe65ef7935";
+const form = document.getElementById("form");
+const currentWeatherContainer = document.getElementById("currentWeather");
+const forecastWeatherContainer = document.getElementById("forecastWeather");
+let city = "";
+let lat = "";
+let lon = "";
+let temp = "";
+let icon = "";
+let wind = "";
+let humidity = "";
+let description = "";
+let tempDate = "";
+let uv = "";
+const renderRecentCities = () => {
+  const container = document.getElementById("recent");
```

Comment:
You have declared this container here using vanilla javascript syntax, but then below on line 19, you have targeted the same div using jQuery, making this line redundant.